Makefile

```diff
@@ -64,6 +68,8 @@ dev:
 	  fi \
 	done;
 
+dev: reinstall dev-only
```

Comment:
It seems `reinstall` is only used by this `dev` target. I suggest removing `reinstall` completely and just using the two targets manually. This coupled with the rename of `dev-only` would give:
```
dev: remove install dependencies
```